src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working.");

```

Comment:
Similarly, the message in the `stopWork` method should indicate the type of machine that has stopped working. Make sure the message clearly states that it is a Bulldozer.